changelog/C/Crown.md

```diff
@@ -0,0 +1,10 @@
+# What Has Changed In the “Crown” Article
+
+## Major changes
```

Comment:
Same comment as the one made on the Shepherd article: The changes here seem sensible and need not be repeated here. An approximate count of the number of changes would be sufficient (no need to rework this file, you can do so on the next articles.)